packages/terra-application-links/src/tabs/_ApplicationTabMenu.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Popup from 'terra-popup';
+import { Switch, Route } from 'react-router-dom';
+import ApplicationTabMenuContent from './_ApplicationTabMenuContent';
+import ApplicationTabUtils from './ApplicationTabUtils';
+import MenuButton from './_MenuButton';
+
+const propTypes = {
+  /**
+   * Child tabs to be placed in the tab menu.
+   */
+  children: PropTypes.node,
+  /**
+   * Should the menu he hidden, set to true if there are no hidden items.
+   */
+  isMenuHidden: PropTypes.bool,
+  /**
+   * Ref callback for menu button.
+   */
+  refCallback: PropTypes.func,
+};
+
+const contextTypes = {
+  /* eslint-disable consistent-return */
+  intl: (context) => {
```

Comment:
Can we use injectIntl instead of interfacing with the context directly? https://github.com/yahoo/react-intl/wiki/API#injectintl